src/lib/components/FieldSelectModel.svelte

```diff
@@ -3,13 +3,23 @@
 
 	import LL from '$i18n/i18n-svelte';
 	import { settingsStore } from '$lib/localStorage';
+	import { getLastUsedModels } from '$lib/sessions';
 
 	import FieldSelect from './FieldSelect.svelte';
 
 	let value: string | undefined = $settingsStore.ollamaModel || '';
 
 	$: disabled = !$settingsStore.ollamaModels.length;
-	$: models = $settingsStore.ollamaModels.map((m) => ({ value: m.name, label: m.name }));
+	$: models = $settingsStore.ollamaModels.map((m) => ({
+		value: m.name,
+		label: m.name,
+		extra: m.details.parameter_size
+	}));
+	$: lastUsedModels = getLastUsedModels().map((m) => ({
+		value: m,
+		label: m,
+		extra: models.find((model) => model.value === m)?.extra
+	}));
```

Comment:
- [ ] If there are no recent models it renders an empty list. If a group doesn't have any results we should include an empty message: `No recently used models`. I thought of suggesting hiding the group entirely but I feel like it's better UX if the user knows the feature even exists before they use a model for the first time.
  ![image](https://github.com/user-attachments/assets/b4119a78-fae2-4560-a9f8-7c5ee3cb5114)

- [ ] The models in the recent list are duplicated in the other list, so when you type a model name you see the result twice. We probably shouldn't include the model in the 2nd group.
  ![image](https://github.com/user-attachments/assets/665917a3-8d67-48fc-890b-fe6653e35c17)

- [ ] Alphabetical sorting got lost in the first group:
  ![image](https://github.com/user-attachments/assets/97bc6734-7904-4dde-9721-b0f94202e6b4)

Reply:
> Alphabetical sorting got lost in the first group

Actually, I did this on purpose. I thought it'd be better to have the recently used models sorted by last used. Then, in the second group, models are sorted alphabetically. Should both groups be sorted alphabetically?

Reply:
Ah, interesting. I'm not sure what the best sorting way should be, I'll have to to play with it.
Let's keep your current sorting by frequency, sounds like a good bet.

Make sure we add a couple of assertions for that sorting logic.

Reply:
Maybe we should reverse the order, in this screenshot you can see 2 of the 3 sessions with `codestral:latest` but `phi:latest` appears first:

![image](https://github.com/user-attachments/assets/d712fed1-65ed-445e-a4e5-95b22ce965f1)

Reply:
I see, for some reason I always pictured this feature with the most popular at the top.
But that's okay, let's leave it with the current order.

Reply:
We could also sort the models by _most used_, but there'd be some details that we should consider:
- we should iterate over the sessions and count how many times was each model used. Then, we could sort the 5 most used models. However, if the user had a large number of sessions, we should see if this still performs well.
- let's suppose that my 5 most used models have ~100 sessions each. If I pulled some new models and preferred to use them from now on, they wouldn't appear in the first group of models until I have more sessions that use those newly pulled models.

Reply:
You are right, there's quite a bit of complexity if we go the _most used_ route.
What we have now is already a huge UX improvement, we can punt these ideas for later.